gnosis/conditional-tokens-explorer
# Details consistency related to omenMarkets and questions

Closes #731 

To test this I've used 
```
REACT_APP_CTE_GRAPH_HTTP_RINKEBY='https://api.thegraph.com/subgraphs/name/davidalbela/conditional-tokens-rinkeby'
REACT_APP_CTE_GRAPH_WS_RINKEBY='wss://api.thegraph.com/subgraphs/name/davidalbela/conditional-tokens-rinkeby'
```

All new tests are ignored because I couldn't make them work properly. After the feedback, I will focus on test the changes on the presentational components only. The problems seem related to de MockProvider that ignores the use of our wrapper around different contexts for Omen and CTE. 

Both components are shown. It is simpler that way as discussed with @mariano-aguero , the data from the hooks is inside a container `OmenMarketsOrQuestions`, we can change that logic from there. Another solution is to move the logic to `Contents` but is not trivial enough to do that, I prefer to maintain a Container that fetches everything needed to make a decision about visibility. 

I wanted to push this with a test because I found that are multiple scenarios in which this looks different according to the data available, and mocked data should be able to do that job.
